src/tigerbeetle/cli.zig

```diff
@@ -377,6 +374,36 @@ pub fn parse_args(allocator: std.mem.Allocator, args_iterator: *std.process.ArgI
             };
         },
         .start => |start| {
+            // Allowlist of stable flags. --development will disable automatic multiversion
+            // upgrades too, but the flag itself is stable.
+            const stable_args = .{
+                "addresses",   "cache_grid",   "positional",
+                "development", "experimental",
+            };
+            inline for (std.meta.fields(@TypeOf(start))) |field| {
+                const stable_field = comptime for (stable_args) |stable_arg| {
+                    if (std.mem.eql(u8, field.name, stable_arg)) {
+                        break true;
+                    }
+                } else false;
+                if (stable_field) continue;
+
+                if (comptime !stable_field) {
```

Comment:
Isn't the `if (stable_field) continue;` unnecessary, since it is followed by `if (comptime !stable_field) {`?

Reply:
Yep!